code/modules/spells/spell_types/pointed/duffelbag_curse.dm

```diff
@@ -10,7 +10,7 @@
 	invocation = "BA'R A'RP!"
 	invocation_type = INVOCATION_SHOUT
 	range = 7
-	cooldown_min = 50
+	cooldown_min = 80
```

Comment:
but why

if you want to nerf the spell's cooldown, why not nerf the spell's maximum cooldown instead, since the spell is most useful as an effective 1 point investment?

Reply:
doesnt the max cost scale of that

Reply:
no, the max cost is defined above